OpenMEEG/CMakeLists.txt

```diff
@@ -185,7 +185,8 @@ add_subdirectory(Wrapping)
 #   Tests
 
 if (BUILD_TESTING)
-    include(CreateTestEnvironment)
+    # XXX : this breaks when trying to build OpenMEEG without the project :(
+    # include(CreateTestEnvironment)
```

Comment:
Obviously not the proper fix.
I'm committing the following instead:

     #   Hdf5 dir
 
     set(TestConfigFile "${CMAKE_BINARY_DIR}/TestConfig.cmake")
-    get_property(HDF5_LIB TARGET hdf5 PROPERTY LOCATION)
-    get_filename_component(HDF5_LIB ${HDF5_LIB} DIRECTORY)
-    get_filename_component(HDF5_ROOT_DIR ${HDF5_LIB} DIRECTORY)
+
+    set(HDF5_ROOT_DIR ${HDF5_LIBRARY_DIRS})
+    if (TARGET hdf5)
+        get_property(HDF5_LIB TARGET hdf5 PROPERTY LOCATION)
+        get_filename_component(HDF5_LIB ${HDF5_LIB} DIRECTORY)
+        get_filename_component(HDF5_ROOT_DIR ${HDF5_LIB} DIRECTORY)
+    endif()

Can you merge (from master) and check this works ??
Out of curiosity, did the tests work. This is made for tests to find the proper libraries. But I think that for windows and mac I'm installing everything in the same install tree, so all of this might be totally obsolete.... For the time being I'm  being conservative....